src/filter.rs

```diff
@@ -13,6 +13,8 @@ pub enum FilterError {
 pub struct RayTracing<'a> {
     handle: OIDNFilter,
     device: &'a Device,
+    albedo: Option<&'a [f32]>,
+    normal: Option<&'a [f32]>,
```

Comment:
So maybe using the same lifetime as the filter is ok for https://github.com/Twinklebear/oidn-rs/issues/5 ? I'm not sure if that introduces a different issue with the device and albedo/normal being marked as having the same life times?

E.g. does some code like the below compile? I forget a bit the bounding/rules of lifetimes now actually, whether marking them all as `'a` is saying none should outlive the device, or tying them to a similar lifetime as the device.

```
let device = oidn::Device::new();
{
    let mut filter = oidn::RayTracing::new(&device);
    let albedo = load_exr(&albedo_exr);
    filter
        .set_srgb(true)
        .set_img_dims(input.width() as usize, input.height() as usize)
        .albedo(&albedo.img[..])
       ... /// etc.
}
```

Reply:
One can actually do `let albedo: EXRData;` I didn't know this at all. I.e. that Rust knows forward declarations to help resolve lifetime issues w/o introducing a `mut` requirement. Neat.
I fixed the README and added a 2nd lifetime for `albedo` & `normal`. New PR incoming. :)